tests/misra/test_misra.sh

```diff
@@ -39,26 +44,28 @@ cppcheck() {
           "$@"
 
   # sanity check the reported coverage
-  no="$(grep '^No ' $report | wc -l)"
-  yes="$(grep '^Yes' $report | wc -l)"
-  echo "$yes checks enabled, $no disabled"
-  if [[ $yes -lt 250 ]]; then
-    echo "Count of enabled checks seems too low."
-    exit 1
-  fi
-  if [[ $no -ne 99 ]]; then
-    echo "Disabled check count threshold doesn't match, update to $no"
-    exit 1
+  if [ $build_dir_exists == false ]; then
```

Comment:
is there some other way to do this sanity check? if not, i'd consider just removing it. you always want your test to run the same every time

Reply:
Hmm in my latest commit I removed it and the report - can restore if you see it necessary. As I see it we already have `--enable=all` and `--force` enabled which forces every necessary checkers to wake up already, so it should be safe. As I see it, checkers are not used for cached files unless the content change, so I can't think of any native way to ensure the same number of checkers everytime with cache enabled